lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp

```diff
@@ -18,11 +18,9 @@
 #include <stdexcept>
 #include <vector>
 
-#ifdef _WIN32
-#define OCL_RUNTIME_EXPORT __declspec(dllexport)
-#else
-#define OCL_RUNTIME_EXPORT
-#endif // _WIN32
+#include "gc_utils.h"
```

Comment:
Need to update the CMakeList.txt to add the directory of newly added "gc_utils.h" to resolve the build error.

Reply:
Yes. Thanks for the advice! I will come back to this PR soon.